docker/Dockerfile.pdo-tp-ccf

```diff
@@ -81,7 +81,7 @@ RUN touch set_no_proxy.sh \
 # Also for configuring ccf network, the container ip address must be added to no_proxy
 ENTRYPOINT `/pdo/private-data-objects/build/common-config.sh -e` \
 	&& cd private-data-objects/ccf_transaction_processor/ \
-        && make config\
```

Comment:
This seems an unintended change?  At least i assume you did not intentionally wanted to make intendation worse?